Client.xcodeproj/project.pbxproj

```diff
@@ -5510,64 +5523,64 @@
 		E4E0BB141AFBC9E4008D6260 /* Shared */ = {
 			isa = PBXGroup;
 			children = (
-				064741F8AE308A64808E6265 /* Shared.strings */,
-				67AF4B8392ADBAA41F1B3AD0 /* HistoryPanel.strings */,
-				9403451493EE3ADAAEFA2EB1 /* Storage.strings */,
-				AA3047F884B6BA22989FD627 /* Search.strings */,
-				21374EEAA2C69079F77A0AC3 /* Today.strings */,
-				05014FF0A3822CA5C103E9C9 /* Menu.strings */,
-				32484895B328A342AED539C8 /* AuthenticationManager.strings */,
-				968645798276FD71A15F01E5 /* Default Browser.strings */,
-				8DFA42FBB7C25AC38ACEEA0D /* PrivateBrowsing.strings */,
-				7401461DBA6DB88F82931350 /* Localizable.strings */,
```

Comment:
@electricRGB I am assuming these are just the changes that XCode did when you added new files and are simply re-arranged? 🤔

Reply:
Yeah. If you look at them, they're re-added in green around there. I also got scared when I saw that. LOL